src/Microsoft.ManagedZLib/src/CtData.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.ManagedZLib;
+
+public class CtData
+{
+    //Freq or Code
+    public ushort Freq { get; set; } // frequency count
+    //Dad or Len
+    public ushort Len { get; set; } // length of bit string
```

Comment:
@jkoritzinsky would there be a perf benefit in changing these to fields instead of properties?:

```suggestion
    public ushort Freq { get; set; } // frequency count
    //Dad or Len
    public ushort Len { get; set; } // length of bit string
```